groups/alteryx-live/ec2_server.tf

```diff
@@ -25,6 +25,13 @@ module "alteryx_server_ec2_security_group" {
       protocol    = "tcp"
       description = "WMI Access"
       cidr_blocks = join(",", local.azure_dc_cidrs)
+    },
+    {
+      from_port   = 5986
+      to_port     = 5986
+      protocol    = "tcp"
+      description = "Ansible Access"
+      cidr_blocks = join(",", local.internal_cidrs, local.concourse_cidrs)
```

Comment:
Can we do the combination within `locals.tf` instead please and reference from there. It makes the code more legible